tasks/check_requirements.yml

```diff
@@ -2,5 +2,5 @@
 
   - name: Check if OS is Ubuntu 16.04 xenial (we do not support others)
```

Comment:
Should we believe the comment "(we do not support others)" or ignore it?

Reply:
Believe it. This originated from a fork of the old version of this that only worked with 12.x and 14.x. The changes I forked are specific to 16.x.

Reply:
then perhaps
```
    failed_when: ansible_distribution != "Ubuntu" and ansible_distribution_version | version_compare('16.04', '>=')

```
should be simply
```
    failed_when: ansible_distribution != "Ubuntu" 
                   and
                 ansible_distribution_major_version == "16"
```
?